sdk/examples/wallet/16_destroy_account_output.rs

```diff
@@ -1,15 +1,15 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! In this example we will try to destroy the first alias there is in the account. This is only possible if possible
-//! foundry outputs have circulating supply of 0.
+//! In this example we will try to destroy the first account output there is in the account. This is only possible if
+//! possible foundry outputs have circulating supply of 0.
```

Comment:
This was wrong already before
```suggestion
//! In this example we will try to destroy the first account output there is in the account.
```